p2p/src/types.rs

```diff
@@ -48,12 +48,22 @@ pub const MAX_LOCATORS: u32 = 20;
 /// How long a banned peer should be banned for
 const BAN_WINDOW: i64 = 10800;
 
-/// The max peer count
-const PEER_MAX_COUNT: u32 = 125;
+/// The max inbound peer count
+const PEER_MAX_INBOUND_COUNT: u32 = 117;
+
+/// The max outbound peer count
+const PEER_MAX_OUTBOUND_COUNT: u32 = 8;
 
 /// min preferred peer count
 const PEER_MIN_PREFERRED_COUNT: u32 = 8;
```

Comment:
Should this one be inbound/outbound specific as well?

Reply:
Removed `PEER_MIN_PREFERRED_COUNT` in favor of `PEER_MIN_PREFERRED_OUTBOUND_COUNT`